app/components/holidays-overview.hbs

```diff
@@ -0,0 +1,88 @@
+<div
+  {{on "mouseenter" (set this "showDetails" true)}}
+  {{on "mouseleave" (set this "showDetails" false)}}
+  {{create-tracked-ref "holiday-counter"}}
+  class="box-sizing group flex select-none items-center rounded-lg p-2 transition duration-200 ease-in-out hover:bg-gray-50"
+>
+  <div class="flex items-center">
```

Comment:
Can this be made a bit smaller (maybe icon and text on 1 line?) and get a less prominent place, maybe at the bottom of the sidebar?